lib/ui/common/app_colors.dart

```diff
@@ -9,6 +9,8 @@ const Color kcVeryLightGrey = Color(0xFFE3E3E3);
 const Color kcBackground = Color(0xFFFFFFFF);
 const Color kcIconBackground = Color(0x4BFFF3CE);
 const Color kcBackgroundColor = kcDarkGreyColor;
+const Color kcLightRed = Color(0xFFCC444B);
```

Comment:
kindly note this is not the approved way of adding colors as it will be removed soon. Kindly add your color to the `AppColors` class and register it in the palette.